src/services/billing-services/load-invoice-last-updated-service.js

```diff
@@ -0,0 +1,46 @@
+import { AxiosHttpClientAdapter, parseHttpResponse } from '../axios/AxiosHttpClientAdapter'
+import graphQLApi from '../axios/makeGraphQl'
+import { makeBillingBaseUrl } from './make-billing-base-url'
+import { formatDateToUSBilling, getCurrentMonthStartEnd } from '@/helpers/convert-date'
+
+export const loadInvoiceLastUpdatedService = async () => {
+  const dateRange = getCurrentMonthStartEnd()
+  const payload = {
+    query: `
+    query getBillDetail {
+      products: billDetail(
+        groupBy: [createdDate]
+        filter: {
+          periodFromRange: {
+            begin: "${dateRange.dateInitial}", end: "${dateRange.dateFinal}"
+          }
+        }
+      ) {
+        last_updated: createdDate
+      }
+    }`
+  }
+
+  let httpResponse = await AxiosHttpClientAdapter.request(
+    {
+      url: `${makeBillingBaseUrl()}`,
+      method: 'POST',
+      body: payload
+    },
+    graphQLApi
+  )
+
+  httpResponse = adapt(httpResponse)
+
+  return parseHttpResponse(httpResponse)
+}
+
+const adapt = (httpResponse) => {
+
+  const { body, statusCode } = httpResponse
+
+  return {
+    body: `Last Updated: ${formatDateToUSBilling(body?.data?.products[0].last_updated)}` ?? [],
+    statusCode
+  }
+}
```

Comment:
Recomendo criar um retorno default ou algo do tipo;
Código de sugestão:  
```suggestion
const adapt = ({ body, statusCode }) => {

  const [bill] = body.data?.products || []
  const dataUpdated = formatDateToUSBilling(bill.last_updated) || '--'
  const formtText = dataUpdated ? `Last Updated: ${dataUpdated}` :  '---'
  return {
    body: formText,
    statusCode
  }
}
```